public/css/globals/_variables.styl

```diff
@@ -5,7 +5,7 @@
 /* Typefaces */
 
 $font-facit = 'jaf-facitweb', 'Helvetica Neue', Helvetica, Arial, sans-serif
-$font-inconsolata = 'inconsolata', menlo, consolas, monospaced
+$font-inconsolata = 'inconsolata', menlo, consolas, monospaced, monospace
```

Comment:
yeah i think this was a typo and monospaced should've been monospace from the start